netlify.toml

```diff
@@ -1,6 +1,7 @@
 [[headers]]
   for = "/*"
   [headers.values]
+    Cache-Control = "public, max-age=3600"
```

Comment:
the scope of the issue is to enable cache for build, 

why is this header added? what will it change?

Reply:
@sambhavgupta0705

Reply:
yes but how is it helping with cache build? this is a setting for the server and client side caching. So imho totally unrelated and probably something we should not add